apps/ima/index.js

```diff
@@ -31,11 +31,145 @@ module.exports = {
       backLink: 'current-progress'
     },
+    '/in-the-uk':{
+      behaviours: SaveFormSession,
+      forks: [
+        {
+          target: '/name',
+          condition: {
+            field: 'in-the-uk',
+            value: 'yes'
+          }
+        },
+        {
+          target: '/cannot-use-form',
+          condition: {
+            field: 'in-the-uk',
+            value: 'no'
+          }
+        }
+      ],
+      fields: ['in-the-uk'],
+      locals: { showSaveAndExit: true },
+      continueOnEdit: true,
+      next: '/name',
+      backLink: 'who-are-you'
+    },
+    
+    '/name':{
+      behaviours: SaveFormSession,
+      fields: ['name'],
+      locals: { showSaveAndExit: true },
+      next: '/is-your-email'
+    },
+    '/is-your-email':{
+      behaviours: SaveFormSession,
+      fields: ['current-email'],
+      forks: [
+        {
+          target: '/phone-number',
+          condition: {
+            field: 'current-email',
+            value: 'yes'
+          }
+        },
+        {
+          target: '/has-email',
+          condition: {
+            field: 'current-email',
+            value: 'no'
+          }
+        }
+      ],
+      locals: { showSaveAndExit: true },
+      continueOnEdit: true,
+      next: '/phone-number',
+      backLink: 'name'
+    },
+    '/phone-number': {
+      behaviours: SaveFormSession,
+      fields: ['phone-number', 'phone-number-details'],
+      locals: { showSaveAndExit: true },
+      next: '/immigration-detention'
+    },
+    '/has-email':{
+      behaviours: SaveFormSession,
+      fields: ['email-address', 'email-address-details'],
+      locals: { showSaveAndExit: true },
+      next:'/phone-number'
+    },
+    '/immigration-detention':{
+      behaviours: SaveFormSession,
+      fields: [
+        'has-address',
+        'house-number',
+        'street',
+        'townOrCity',
+        'county',
+        'postcode'
+      ],
+      locals: { showSaveAndExit: true },
+      next: '/medical-record'
+    },
+    '/cannot-use-form':{},
+   
+    '/someone-else':{
+      behaviours: SaveFormSession,
+      fields:[
+        'someone-else-fullname',
+        'someone-else-relationship',
+        'someone-else-organisation'
+      ],
+      continueOnEdit: true,
+      locals: { showSaveAndExit: true },
+      next:'/in-the-uk'
+    },
+    '/legal-representative-details':{
+      behaviours: SaveFormSession,
+      fields: [
+        'legal-representative-fullname',
+        'legal-representative-organisation',
+        'legal-representative-house-number',
+        'legal-representative-street',
+        'legal-representative-townOrCity',
+        'legal-representative-county',
+        'legal-representative-postcode',
+        'legal-representative-phone-number',
+        'is-legal-representative-email',
+        'legal-representative-email'
+              ],
+      continueOnEdit: true,
+      locals: { showSaveAndExit: true },
+      next:'/in-the-uk'
+    },
```

Comment:
I would move these two steps before the '/in-the-uk' step so readability of this file matches the figma as much as possibile

Reply:
Changes done